src/coordinator.rs

```diff
@@ -64,7 +64,7 @@ impl Coordinator {
                 .store_from_workspace(job, workspace)
                 .context("could not store job output")?;
         } else {
-            log::debug!("already had output of this job; skipping");
+            log::debug!("already had output of job {}; skipping", &job);
```

Comment:
why does here need `&` but line 55 not?

Reply:
huh, looks like it doesn't actually need it! That's weird; usually Clippy points out unnecessary borrows!